tests/integration/ceph_upgrade_test.go

```diff
@@ -326,16 +310,7 @@ func (s *UpgradeSuite) verifyFilesAfterUpgrade(fsName, newFileToWrite, messageFo
 	}
 }
 
-// UpgradeToV1_3 performs the steps necessary to upgrade a Rook v1.2 cluster to v1.3. It does not
-// verify the upgrade but merely starts the upgrade process.
-func (s *UpgradeSuite) upgradeToV1_3() {
-	require.NoError(s.T(), s.k8sh.ResourceOperation("apply", upgradeManifestTo1_3(s.namespace)))
-
-	require.NoError(s.T(),
-		s.k8sh.SetDeploymentVersion(installer.SystemNamespace(s.namespace), operatorContainer, operatorContainer, installer.Version1_3))
-}
-
-// UpgradeToMaster performs the steps necessary to upgrade a Rook v1.2 cluster to master. It does not
+// UpgradeToMaster performs the steps necessary to upgrade a Rook v1.4 cluster to master. It does not
 // verify the upgrade but merely starts the upgrade process.
 func (s *UpgradeSuite) upgradeToMaster() {
 	require.NoError(s.T(), s.k8sh.DeleteResource("ClusterRole",
```

Comment:
Does this list of resources to delete not need to be updated? I'd be surprised if the list was the same now.

Reply:
I don't think anything needs to be deleted now.... There is no `update-from-v1.4-delete.yaml`

Reply:
I removed this, because I don't think anything needs to be deleted moving from v1.4 to master now.